compiler/circle-interpreter-test/README.md

```diff
@@ -0,0 +1,16 @@
+# circle-interpreter-test
+
+`circle-interpreter-test` executes `circle-interpreter` and validates its results with tflite model's output. 
```

Comment:
IMHO, this test would be better to check valid/invalid input arguments.

Reply:
I tried to check the input arguments' validity through `one-cmds` test, via `one-infer`. Do you think it's better to check it here, and do something else in `one-infer` test? (I have no idea what to do for `one-infer` test if verifying arguments are done here.)

Reply:
If so, how about adding some comments about argument validity test?

Reply:
I added it to README :)